packages/expo-status-bar/src/StatusBar.tsx

```diff
@@ -2,10 +2,23 @@ import React from 'react';
 import {
   Appearance,
   StatusBar as NativeStatusBar,
+  Platform,
   useColorScheme,
   type ColorSchemeName,
   type ColorValue,
 } from 'react-native';
+import { isEdgeToEdge } from 'react-native-is-edge-to-edge';
+
+let shouldWarnAboutEdgeToEdge = Platform.OS === 'android' && isEdgeToEdge();
+
+function potentiallyWarnAboutEdgeToEdge() {
+  if (shouldWarnAboutEdgeToEdge) {
+    shouldWarnAboutEdgeToEdge = false; // warn once
+    console.warn(
+      'Avoid using expo-status-bar with edge-to-edge, as it may cause interference. Instead, use the SystemBars component from react-native-edge-to-edge.'
```

Comment:
```suggestion
      'Using expo-status-bar in apps with edge-to-edge layout enabled may cause unexpected behavior. Instead, use the SystemBars component from react-native-edge-to-edge. Learn more: (insert short link here, such as expo.fyi/edge-to-edge or something)'
```

how about something like this? i always like to add a link also so people have an immediately actionable next step

Reply:
@brentvatne would you like to have a markdown file on the repository, or a link to https://github.com/zoontek/react-native-edge-to-edge#systembars is enough?

Reply:
ah sorry i forgot to respond here. ideally we could create something on expo.fyi that contextualizes this specifically for expo-status-bar and then from there it links out to the edge-to-edge repo - that makes it more resilient to the anchor changing as well (since we can update expo.fyi easily by just changing the markdown file at https://github.com/expo/fyi)